python_modules/libraries/dagster-aws/dagster_aws/ecs/container_context.py

```diff
@@ -1,25 +1,36 @@
-from typing import TYPE_CHECKING, Any, List, Mapping, NamedTuple, Optional
+from typing import TYPE_CHECKING, Any, Dict, List, Mapping, NamedTuple, Optional
 
 from dagster import Array, Field, Noneable, Shape, StringSource
 from dagster import _check as check
 from dagster.config.validate import process_config
 from dagster.core.errors import DagsterInvalidConfigError
 from dagster.core.storage.pipeline_run import PipelineRun
+from dagster.core.utils import parse_env_var
 from dagster.utils import merge_dicts
 
 from ..secretsmanager import get_tagged_secrets
 
 if TYPE_CHECKING:
     from . import EcsRunLauncher
 
+# Config shared between EcsRunLauncher and EcsContainerContext
+SHARED_ECS_SCHEMA = {
+    "env_vars": Field(
+        [StringSource],
```

Comment:
What's the current environment? Like from the container of the task that's launching it?

Is this to maintain parity with how we set env vars in other launchers? Otherwise, we might want to more directly mirror the standard ECS configuration since we've been bitten a bit in the past (secrets) with confused users when we tried to add clever abstractions on top of the raw config:

https://docs.aws.amazon.com/AmazonECS/latest/developerguide/taskdef-envfiles.html

Reply:
Yeah, it's to have a consistent way to set env vars across all three environments

Reply:
in this case, i think its syntactic sugar on top of the exact same functionality (a name and a value). I actually originally had "environment" mapping directly onto the field in the container definition, but decided against it for consistency across the agents and because the raw AWS format didn't offer anything new, unlike secrets

we could additionally offer a separate environment_files key if we wanted (and I would still like to offer a raw JSON config option as an escape hatch)